src/components/data/LocalVariablesContent.js

```diff
@@ -179,7 +225,29 @@ const LocalVariablesContent = ({ workspace, workspace: { workspace: { googleProj
                     value: editType,
                     onChange: ({ value }) => setEditType(value),
                     options: ['string', 'number', 'boolean', 'string list', 'number list', 'boolean list']
-                  }),
+                  })
+                ]) :
+                div({ className: 'hover-only' }, [])
```

Comment:
What is this for?

Reply:
There were warnings about a dangling comma on that line in the Chrome JavaScript console and removing that comma silenced those warnings.

Reply:
I meant line 227 (the empty div).

Reply:
Ah - got it.  I removed the empty div and replaced it with a call to renderDataCell, as is done for the 'Key' column.